applications/CoSimulationApplication/tests/test_CoSimulationApplication.py

```diff
@@ -88,6 +88,7 @@ def AssembleTestSuites():
     # For very long tests that should not be in nighly and you can use to validate
     validationSuite = suites['validation']
     validationSuite.addTests(KratosUnittest.TestLoader().loadTestsFromTestCases([TestCoSimulationCases]))
+    validationSuite.addTest(TestMokFSI('test_mok_fsi_jacobi'))
```

Comment:
@VeronikaSinger can you please revert my changes in the testing since you have a better test? Thx!

Reply:
Done - I deleted the test.